### PR TITLE
FEAT: Events Page

### PR DESCRIPTION
**Pull request by alexkocharyan01**

### CODE REVIEW

File: src/pages/events/index.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                    className="w-full xl:hidden xl:pl-0"
+                    onClick={() => hendelShowMore(item.id)}
+                  >
+                    Read All
+                  </Button>
+                )}
+              </div>
+              {!item.isButtonVisuable ? (
+                <div>
+                  {showMore && item.id === showModeId ? (
```
Comment (DavGalstyan98) [inline src/pages/events/index.tsx:112]: I think you can have 1 button and check classNames and Text inside button component

File: src/pages/events/index.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            </p>
+          </div>
+          <div className="hidden xl:flex">
+            <p className="text-base text-black-light">Home / Events</p>
+          </div>
+        </div>
+        {data.map((item) => (
+          <div
+            key={item.id}
+            className="hover:rounded-2.5 mt-5 flex flex-col rounded border border-gray-thin p-5 hover:border-primary xl:mt-[60px] xl:flex-row xl:items-start xl:justify-between"
```
Comment (rikarsen) [inline src/pages/events/index.tsx]: xl:mt-15

File: src/pages/events/index.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+              </div>
+              {!item.isButtonVisuable ? (
+                <div>
+                  <Button
+                    onClick={() =>
+                      handleShowMore(item.id, showMoreInfo(item.id) ? 'Close' : 'Open')
+                    }
+                    variant={showMoreInfo(item.id) ? 'outlined' : 'text'}
+                    rightIcon={showMoreInfo(item.id) ? null : <RightIcon fill="fill-primary" />}
+                    className={`hidden pl-0  xl:mt-[30px] xl:flex ${
```
Comment (rikarsen) [inline src/pages/events/index.tsx]: xl:mt-7.5

File: src/pages/events/index.tsx [hunk 4]
```diff
@@ -67,7 +58,7 @@ const News = () => {
             key={item.id}
             className="hover:rounded-2.5 mt-5 flex flex-col rounded border border-gray-thin p-10 hover:border-primary xl:mt-10 xl:flex-row xl:items-start xl:justify-between"
           >
-            <div className="xl:w-[35%]">
+            <div className="xl:w-[500px]">
```
Comment (Arthur02V) [inline src/pages/events/index.tsx:61]: maybe max-width

File: src/pages/events/index.tsx [hunk 5]
```diff
@@ -111,9 +102,9 @@ const News = () => {
               </div>
               {!item.isButtonVisible ? (
                 <div>
-                  {showMoreInfo(item.id) ? (
+                  {item.id === showModeId ? (
                     <Button
-                      onClick={() => setShowMore(false)}
+                      onClick={() => setShowModeId(0)}
```
Comment (Arthur02V) [inline src/pages/events/index.tsx:107]: Id 0 can be a thing null should be the value

**Review by Arthur02V — CHANGES_REQUESTED** [review]

File: src/pages/events/index.tsx [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+      return true
+    }
+
+    return false
+  }
+
+  return (
+    <Container className="xl:pb=[182px] pb-[152px]">
+      <div className="border-b border-gray-thin">
+        <div className="mt-5 flex w-full justify-between border-b border-gray-thin pb-5 xl:pb-10">
```
[the remark this answers is not in the document]
Comment (rikarsen) [reply src/pages/events/index.tsx:45]: and add description inside PageHeader component with condition

File: src/pages/events/index.tsx [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+                <p className="text-base text-black-light">{item.secondInfo}</p>
+              ) : null}
+              <p className="text-base text-black-light xl:mt-2.5">
+                {item.id === showModeId ? item.info : `${item.info.substring(0, 130)}`}
+              </p>
+              <p className="mt-5 text-sm text-black-light xl:hidden">{item.time}</p>
+              <div className="mb-5 mt-5 xl:hidden">
+                {item.id === showModeId ? (
+                  <Button
+                    onClick={() => setShowModeId(0)}
```
Comment (rikarsen) [inline src/pages/events/index.tsx:88]: null instead of 0

**Review by ArthurO2002 — DISMISSED** [review]

**Review by ArthurO2002 — APPROVED** [review]

File: src/pages/events/index.tsx [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+              </div>
+            </div>
+            <div className="xl:flex-1">
+              <h2 className="mt-5 text-xl text-black xl:mt-0 xl:text-2xl">{item.time}</h2>
+              <p className="mt-2.5 text-lg text-black">{item.title}</p>
+              {item.secondInfo && item.id === showModeId ? (
+                <p className="text-base text-black-light">{item.secondInfo}</p>
+              ) : null}
+              <p className="text-base text-black-light xl:mt-2.5">
+                {item.id === showModeId ? item.info : `${item.info.substring(0, 130)}`}
```
Comment (ArthurO2002) [inline src/pages/events/index.tsx:78]: add todo to change later

**Review by rikarsen — APPROVED** [review]

**Pull request merged**